### ISSUE TITLE
U-Boot install/deploy error for missing itc

### ISSUE THREAD

**Issue by lorenzo-arena** [linux-sunxi/meta-sunxi#329]
I'm trying to build a simple `core-image-base` on hardknott for the pine A64 using the `pine64-plus` machine. While building U-Boot I'm getting some errors in the `do_deploy` and `do_install` tasks, related to the missing `u-boot.itb` file:

Log from `do_deploy`:

```
DEBUG: Executing python function sstate_task_prefunc
DEBUG: Python function sstate_task_prefunc finished
DEBUG: Executing shell function do_deploy
install: cannot stat '/mnt/disk2/Projects/experiments/kas/meta-custom/build/tmp/work/pine64_plus-poky-linux/u-boot/1_2021.01-r0/build/u-boot.itb': No such file or directory
WARNING: exit code 1 from a shell command.
ERROR: ExecutionError('/mnt/disk2/Projects/experiments/kas/meta-custom/build/tmp/work/pine64_plus-poky-linux/u-boot/1_2021.01-r0/temp/run.do_deploy.1317347', 1, None, None)
```

Log from `do_install`:

```
DEBUG: Executing python function extend_recipe_sysroot
NOTE: Direct dependencies are ['/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/quilt/quilt-native_0.66.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/patch/patch_2.7.6.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/flex/flex_2.6.4.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/swig/swig_3.0.12.bb:do_populate_sysroot', '/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/gcc/gcc-runtime_10.2.bb:do_populate_sysroot', '/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/gcc/gcc-cross_10.2.bb:do_populate_sysroot', '/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-core/glibc/glibc_2.33.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/pseudo/pseudo_git.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-kernel/dtc/dtc_1.6.0.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-extended/bc/bc_1.07.1.bb:do_populate_sysroot', '/mnt/disk2/Projects/experiments/kas/meta-custom/meta-sunxi/recipes-bsp/atf/atf-sunxi_git.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/python/python3_3.9.5.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/bison/bison_3.7.5.bb:do_populate_sysroot', '/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-kernel/kern-tools/kern-tools-native_git.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-connectivity/openssl/openssl_1.1.1l.bb:do_populate_sysroot', 'virtual:native:/mnt/disk2/Projects/experiments/kas/meta-custom/poky/meta/recipes-devtools/python/python3-setuptools_54.1.1.bb:do_populate_sysroot']
NOTE: Installed into sysroot: []
NOTE: Skipping as already exists in sysroot: ['quilt-native', 'patch-native', 'flex-native', 'swig-native', 'gcc-runtime', 'gcc-cross-aarch64', 'glibc', 'pseudo-native', 'dtc-native', 'bc-native', 'atf-sunxi', 'python3-native', 'bison-native', 'kern-tools-native', 'openssl-native', 'python3-setuptools-native', 'automake-native', 'libtool-native', 'autoconf-native', 'attr-native', 'texinfo-dummy-native', 'gettext-minimal-native', 'm4-native', 'libpcre-native', 'pkgconfig-native', 'libgcc', 'linux-libc-headers', 'gnu-config-native', 'libmpc-native', 'binutils-cross-aarch64', 'zlib-native', 'mpfr-native', 'gmp-native', 'xz-native', 'readline-native', 'sqlite3-native', 'libnsl2-native', 'libtirpc-native', 'gdbm-native', 'libffi-native', 'util-linux-native', 'bzip2-native', 'ncurses-native', 'libcap-ng-native', 'libpcre2-native', 'util-linux-libuuid-native']
DEBUG: Python function extend_recipe_sysroot finished
DEBUG: Executing shell function do_install
install: cannot stat '/mnt/disk2/Projects/experiments/kas/meta-custom/build/tmp/work/pine64_plus-poky-linux/u-boot/1_2021.01-r0/build/u-boot.itb': No such file or directory
WARNING: exit code 1 from a shell command.
ERROR: ExecutionError('/mnt/disk2/Projects/experiments/kas/meta-custom/build/tmp/work/pine64_plus-poky-linux/u-boot/1_2021.01-r0/temp/run.do_install.1314697', 1, None, None)
```

Should't `UBOOT_BINARY` be set to "u-boot-sunxi-with-spl.bin" since that's what we use?

**Comment by nandra** [comment]
@lorenzo-arena should be fixed by #330? Thanks.

**Comment by lorenzo-arena** [comment]
Yes! I'm closing this, thank you.
